pkg/crc/machine/start.go

```diff
@@ -112,12 +112,57 @@ func (client *client) updateVMConfig(startConfig types.StartConfig, vm *virtualM
 	return nil
 }
 
-func growRootFileSystem(sshRunner *crcssh.Runner, preset crcPreset.Preset, persistentVolumeSize int) error {
-	rootPart, err := getrootPartition(sshRunner, preset)
+func restartHost(ctx context.Context, vm *virtualMachine, sshRunner *crcssh.Runner) error {
+	if err := vm.Driver.Stop(); err != nil {
+		return errors.Wrap(err, "Error re-starting machine at stop")
+	}
+	if err := startHost(ctx, vm); err != nil {
+		return errors.Wrap(err, "Error re-starting machine at start")
+	}
+	logging.Debug("Waiting until ssh is available")
+	if err := sshRunner.WaitForConnectivity(ctx, 300*time.Second); err != nil {
+		return errors.Wrap(err, "Failed to connect to the CRC VM with SSH -- virtual machine might be unreachable")
+	}
+	return nil
+}
+
+func ocpGetPVShiftSize(diskSize int, pvSize int) int {
+	defaultPvSize := constants.GetDefaultPersistentVolumeSize(crcPreset.OpenShift)
+	if pvSize > defaultPvSize {
+		return (diskSize - constants.DefaultDiskSize) - (pvSize - defaultPvSize)
+	}
+	return diskSize - constants.DefaultDiskSize
+}
+
+func growRootFileSystem(ctx context.Context, startConfig types.StartConfig, vm *virtualMachine, sshRunner *crcssh.Runner) error {
+	if startConfig.Preset == crcPreset.OpenShift {
+		sizeToMove := ocpGetPVShiftSize(startConfig.DiskSize, startConfig.PersistentVolumeSize)
+		_, _, err := sshRunner.RunPrivileged("move topolvm partition to end of disk", fmt.Sprintf("echo '+%dG,' | sudo sfdisk --move-data /dev/vda -N 5 --force", sizeToMove))
+		var exitErr *ssh.ExitError
+		if err != nil {
+			if !errors.As(err, &exitErr) {
+				return err
+			}
+			if exitErr.ExitStatus() != 1 {
+				return err
+			}
```

Comment:
yes we need to ignore it, after moving the partition it returns exit code 1 (i think because of `Re-reading the partition table failed.: Device or resource busy` but couldn't find anything about exit codes in `sfdisk` man page)